rest/rest-sqs-testing-amazon-java-sdk/src/test/scala/org/elasticmq/rest/sqs/AmazonJavaSdkTestSuite.scala

```diff
@@ -894,6 +895,26 @@ class AmazonJavaSdkTestSuite extends FunSuite with Matchers with BeforeAndAfter
     msgIdSent should be(msgIdReceived)
   }
 
+  test("should create queue with redrive policy.") {
+    // When
+    val queueUrl = client.createQueue(new CreateQueueRequest("testQueue1")
+      .withAttributes(Map(
+        defaultVisibilityTimeoutAttribute -> "1",
+        redrivePolicyAttribute ->
+          """
+            |{
+            |  "queueName":"dlq1",
```

Comment:
Keeping compatibility with SQS, shouldn't this be called `deadLetterTargetArn`? Also, the SQS docs (I'm looking here: http://docs.aws.amazon.com/AWSSimpleQueueService/latest/SQSDeveloperGuide/sqs-dead-letter-queues.html), don't mention the ability to specify e.g. the `defaultVisibilityTimeout`.